addons/base_vat/models/res_partner.py

```diff
@@ -154,7 +154,8 @@ def check_vat(self):
             if not partner.vat:
                 continue
 
-            if company.vat_check_vies and partner.commercial_partner_id.country_id in eu_countries:
+            is_eu_country = partner.commercial_partner_id.country_id in eu_countries
+            if company.vat_check_vies and is_eu_country and partner.company_type == "company":
```

Comment:
```suggestion
            if company.vat_check_vies and is_eu_country and partner.is_company:
```

sorter and more readable imo